I2T2/io.py

```diff
@@ -16,7 +16,7 @@
 import scipy.io as spio
 
 # Cell
-def load_dcm(path_to_dicom_dir=None, sort_by_slice_location=True):
+def load_dcm(path_to_dicom_dir=None, sort_by_image_position_patient=True, dicom_extension='.dcm'):
```

Comment:
Could you also modify the docstring so the documentation is correct in the website? 
- replace `sort_by_slice_location` by `sort_by_image_position_patient`
- add `dicom_extension`